### ISSUE TITLE
Build failure: Could not find com.parashift.amp:amp-plugin:1.1.1

### ISSUE THREAD

**Issue by andreaskring** [cetra3/onlyoffice-alfresco#32]
I have checked out release 1.4.1, but the project fails to build when I do this (from the project root):
```
$ git checkout 1.4.1
$ cd repo
$ gradle amp

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'onlyoffice-alfresco-share'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find com.parashift.amp:amp-plugin:1.1.1.
     Searched in the following locations:
         https://mvn.parashift.com.au/public/com/parashift/amp/amp-plugin/1.1.1/amp-plugin-1.1.1.pom
         https://mvn.parashift.com.au/public/com/parashift/amp/amp-plugin/1.1.1/amp-plugin-1.1.1.jar
     Required by:
         project :
```

So it seems as if `com.parashift.amp:amp-plugin:1.1.1` is not available?

(and thanks for a great module by the way!)
/Andreas

**Comment by cetra3** [comment]
It should work if you change the maven repo to jcenter as it's also hosted there.

**Comment by Dnhatsave** [comment]
I @andreaskring this is working for you?
i've change the repository to jcenter but i now i got 100 erros of JavaCompile as mentioned by @sergefedorow  on #22 .

**Comment by andreaskring** [comment]
@Dnhatsave Yes - it is working for me. You can see the fix in PR #33

**Comment by cetra3** [comment]
I have resolved the build issues and uploaded a new version.
